index.html

```diff
@@ -8,10 +8,21 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
+  <link rel='stylesheet' type='text/css' href='http://fonts.googleapis.com/css?family=Roboto'>
 </head>
 
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div name="tech" class="m-left-10">
```

Comment:
divs don't use the name attribute, you should add this as a class. the `m-left-10` although is a good practice, we'll add it when we create the layout. That way we'll verify that `10` is really what we want.